src/Icicle/Avalanche/Eval.hs

```diff
@@ -338,9 +338,9 @@ evalStmt evalPrim now xh values bubblegum ah stmt
     Output n t xts
      -> do  vs  <- traverse ((baseValue =<<) . eval . fst) xts
             case (vs, unmeltValue vs t) of
-              (v:[], _)    -> return (ah, [(n, v)])
-              (_, Just v)  -> return (ah, [(n, v)])
-              (_, Nothing) -> Left (RuntimeErrorOutputTypeMismatch n t vs)
+              (_,    Just v)  -> return (ah, [(n, v)])
+              (v:[], Nothing) -> return (ah, [(n, v)])
```

Comment:
would like a short comment.. why would this happen? "the output doesn't match the type, but there's only one output so we might as well return it"